app/models/period.rb

```diff
@@ -92,6 +92,10 @@ def bp_control_range_end_date
     bp_control_range.end.to_s(:day_mon_year)
   end
 
+  def ltfu_since_date
```

Comment:
💯

Reply:
I've been meaning to revisit this name, `ltfu_since_date` is a little misleading. It suggests the patient has been ltfu since 12 months but they're actually ltfu since the `period`. They haven't _had a BP since 12 months ago_. 

Any ideas?

Reply:
I don't really have anything better for this right now.